### DISCUSSION TITLE
Best practices hosting elsewhere without docker

### DISCUSSION THREAD

**Discussion by emanet29** [serverpod/serverpod#2116] [Q&A] [answered]
Before I get started, do you know the procedure to install serverpod without Docker on Ubuntu. Dart and Postgre are well installed. Is it then enough to upload the files from the _server folder? Thanks

**Comment by Isakdl** [comment, marked as answer]
We have a PR pending for the documentation discussing this in more detail (WIP)

https://github.com/serverpod/serverpod_docs/pull/87

For deploying to a production/hosted environment check out the current docs: https://docs.serverpod.dev/deployments/deployment-strategy
